mailpoet/assets/css/src/components-public/_public.scss

```diff
@@ -63,6 +63,12 @@ $form-line-height: 1.4;
   .mailpoet-has-font-size {
     line-height: $form-line-height;
   }
+
+  fieldset {
```

Comment:
I tried creating a testing form and found a problematic case. I guess it's caused by my specific WP theme ([Storefront](https://pcm.wordpress.org/themes/storefront/)), but our forms could look bad. Here is [an example](https://d.pr/i/Aj81Nc). In my opinion, we should reset the background color for the fieldset and the legend tag. Do you agree?

Reply:
Nice catch! Thank you! I agree.